.docker/images/govcms7/scripts/govcms-deploy

```diff
@@ -6,46 +6,39 @@
 # Ensure tmp folder always exists.
 mkdir -p /app/sites/default/files/private/tmp/
 
+# Database updates, cache rebuild.
+common_deploy () {
+    drush updb -y
+    drush cc all -y
+    
+    # Non production environments.
+    if [[ ! -z "$LAGOON_ENVIRONMENT_TYPE" && "$LAGOON_ENVIRONMENT_TYPE" != "production" ]]; then
+        # Enable stage file proxy post db-import.
+        drush en stage_file_proxy -y
+    fi
+}
+
 # Non production environments.
 if [[ "$LAGOON_ENVIRONMENT_TYPE" != "production" ]]; then
```

Comment:
Do we need to check if the $LAGOON_ENVIRONMENT exist? 
For example, 
if [[ ! -z "$LAGOON_ENVIRONMENT_TYPE" && "$LAGOON_ENVIRONMENT_TYPE" != "production" ]]; then

Reply:
@AMDS for your interest, for Drupal 8 there is a further improved version of this script coming in another PR.
https://github.com/simesy/scaffold-tooling/blob/master/scripts/govcms-deploy